examples/webgl_loader_fbx.html

```diff
@@ -0,0 +1,203 @@
+
+					geometry.vertices.push( new THREE.Vector3( - size, - 0.04, i ) );
+					geometry.vertices.push( new THREE.Vector3(   size, - 0.04, i ) );
+
+					geometry.vertices.push( new THREE.Vector3( i, - 0.04, - size ) );
+					geometry.vertices.push( new THREE.Vector3( i, - 0.04,   size ) );
+
+				}
+
+				var line = new THREE.LineSegments( geometry, material );
+				scene.add( line );
+
+                //////////////////////////////////////////////////////////////
+				// Lights
+                scene.add( new THREE.AmbientLight( 0xcccccc ) );
+
+                //////////////////////////////////////////////////////////////
+                // stats
+				stats = new Stats();
+				stats.domElement.style.position = 'absolute';
+				stats.domElement.style.top = '0px';
+                container.appendChild( stats.domElement );
+
+
+                //////////////////////////////////////////////////////////////
+				// model
+				var manager = new THREE.LoadingManager();
+				manager.onProgress = function ( item, loaded, total ) {
+
+					console.log( item, loaded, total );
+
+				};
+
+				var onProgress = function ( xhr ) {
+					if ( xhr.lengthComputable ) {
+						var percentComplete = xhr.loaded / xhr.total * 100;
+						console.log( Math.round(percentComplete, 2) + '% downloaded' );
+					}
+				};
+
+				var onError = function ( xhr ) {
+				};
+
+                var loader = new THREE.FBXLoader( manager );
+				loader.load( 'models/fbx/xsi_man_skinning.fbx', function ( object ) {
+
+					object.traverse( function ( child ) {
+
+						if ( child instanceof THREE.Mesh ) {
+
+							// pass
+
+                        }
+
+						if ( child instanceof THREE.SkinnedMesh ) {
+
+                            if ( child.geometry.animations !== undefined || child.geometry.morphAnimations !== undefined ) {
+
+                                child.mixer = new THREE.AnimationMixer( child );
+                                mixers.push( child.mixer );
+
+                                var action = child.mixer.clipAction( child.geometry.animations[0] );
+                                action.play();
+
+                            }
+
+
+                        }
+
+
+					} );
+                    scene.add( object );
+
+
+                }, onProgress, onError );
+
+                //////////////////////////////////////////////////////////////
+				renderer = new THREE.WebGLRenderer();
+				renderer.setPixelRatio( window.devicePixelRatio );
+                renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.setClearColor( 0x000000 );
+                container.appendChild( renderer.domElement );
+
+                //////////////////////////////////////////////////////////////
+                // controls, camera
+                controls = new THREE.OrbitControls( camera, renderer.domElement );
+                controls.target.set( 0, 12, 0 );
+				camera.position.set( 2, 18, 28 );
+				camera.lookAt( new THREE.Vector3( 0, 15, 0 ) );
```

Comment:
`lookAt()` should not be required, but if you call it, then call `lookAt( controls.target )`.